### PR TITLE
update(epp): vtom rest api proc

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Target serie

- [X] 20.04.x
- [X] 20.10.x
- [X] 21.04.x
- [X] 21.10.x
- [X] 22.04.x (master)

### CODE REVIEW

**Review by lchrdn — DISMISSED** [review]

**Review by Sims24 — APPROVED** [review]

File: fr/integrations/plugin-packs/procedures/applications-vtom-restapi.md [hunk 1]
```diff
@@ -1,35 +1,177 @@
...
+
+Le Pack supporte les méthodes d'authentification:
+* par utilisateur et mot de passe
+* par token directement
+
+La version minimum VTOM 6.6.1a est nécessaire pour le bon fonctionnement du Pack:
+* /auth/1.0/authorize
+* /monitoring/1.0/jobs/status 
+
+Pour les versions antérieurs, il est nécessaire d'utiliser le mode **legacy** du Plugin.
```
Comment (lchrdn) [inline fr/integrations/plugin-packs/procedures/applications-vtom-restapi.md:58]:
```suggestion
Pour les versions antérieures, il est nécessaire d'utiliser le mode **legacy** du Plugin.
```

File: en/integrations/plugin-packs/procedures/applications-vtom-restapi.md [hunk 2]
```diff
@@ -1,35 +1,178 @@
...
+
+## Configuration
+
+### Host
+
+* Log into Centreon and add a new Host through **Configuration > Hosts**
+* Fill the **Name**, **Alias** & **IP Address / DNS** fields according to your **VTOM Rest API** server settings
+* Select the **App-Vtom-Restapi-custom** template to apply to the Host
+
+> Once the template applied, some Macros have to be configured:
```
Comment (lchrdn) [inline en/integrations/plugin-packs/procedures/applications-vtom-restapi.md:100]:
```suggestion
> Once the template is applied, some Macros have to be configured:
```

File: fr/integrations/plugin-packs/procedures/applications-vtom-restapi.md [hunk 3]
```diff
@@ -1,35 +1,177 @@
...
+
+```bash
+yum install centreon-plugin-Applications-Vtom-Restapi
+```
+
+2. Sur l'interface Web de Centreon, installer le Pack **VTOM Rest API** depuis la page **Configuration > Packs de plugins**.
+
+<!--Offline License-->
+
+1. Installer le Plugin Centreon sur tous les collecteurs Centreon devant superviser des resources **VTOM Rest API** :
```
Comment (lchrdn) [inline fr/integrations/plugin-packs/procedures/applications-vtom-restapi.md:76]:
```suggestion
1. Installer le Plugin Centreon sur tous les collecteurs Centreon devant superviser des ressources **VTOM Rest API** :
```

File: fr/integrations/plugin-packs/procedures/applications-vtom-restapi.md [hunk 4]
```diff
@@ -1,35 +1,177 @@
...
+3. Sur l'interface Web de Centreon, installer le Pack **VTOM Rest API** depuis la page **Configuration > Packs de plugins**.
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configuration
+
+### Hôte
+
+* Ajoutez un Hôte à Centreon depuis la page **Configuration > Hôtes**
+* Complétez les champs **Nom**, s**Alias** & **IP Address / DNS** correspondant à votre serveur **VTOM Rest API**.
```
Comment (lchrdn) [inline fr/integrations/plugin-packs/procedures/applications-vtom-restapi.md:97]:
```suggestion
* Complétez les champs **Nom**, **Alias** & **IP Address / DNS** correspondant à votre serveur **VTOM Rest API**.
```

**Comment by Sims24** [comment]
Will be updated to v2

**Pull request closed without merging**